buildenv/docker/mkdocker.sh

```diff
@@ -644,6 +715,24 @@ done
   echo " && rm -f jdk*.tar.gz"
 }
 
+install_criu() {
+  echo ""
+  echo "# Install CRIU and set capabilities."
+  echo "RUN cd /tmp \\"
+  echo " && $wget_O v${criu}.tar.gz https://github.com/checkpoint-restore/criu/archive/refs/tags/v${criu}.tar.gz \\"
+  echo " && tar -xzf v${criu}.tar.gz \\"
+  echo " && cd criu-${criu} \\"
+  echo " && make \\"
+  echo " && make install \\"
+  echo " && cd .. \\"
+  echo " && rm -rf v${criu}.tar.gz criu-${criu} \\"
```

Comment:
Please don't include the version in the local archive name (just use `criu.tar.gz`, following the pattern for other packages).